core/src/main/java/com/linecorp/armeria/server/http/auth/HttpAuthServiceBuilder.java

```diff
@@ -30,73 +32,53 @@
 import com.linecorp.armeria.server.Service;
 
 /**
- * Builds a new {@link HttpAuthService}.
+ * Builds a new {@link HttpAuthService}. {@link Authorizer}s are executed without short-circuiting -
+ * all authorizers will be executed regardless of the result of any individual one.
```

Comment:
Hmm, really? I thought an authorizer returning true can finish the authorization process. Am I missing something?